src/ukrdc_cupid/core/investigate/models.py

```diff
@@ -0,0 +1,66 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship, Mapped
+from sqlalchemy import (
+    MetaData,
+    Column,
+    Integer,
+    Boolean,
+    String,
+    DateTime,
+    Text,
+    ForeignKey,
+    UniqueConstraint,
+    Table,
+    text
+)
+
+from datetime import datetime
+from typing import List, Optional
+
+metadata = MetaData()
+Base = declarative_base(metadata=metadata)
+
+GLOBAL_LAZY = "dynamic"
+
+# Many to many relationship exists between pateint ids 
+# (internal such as ukrdc and external such as NHS)
+# this table allows a issue_id to allow multiple patientids to be appended to any particular issue 
+# similarly it makes it easy to look up which issues are already accosiated with a particular id.
+
+association_table = Table(
+    "patientidtoissue", 
+    Base.metadata, 
+    Column("patient_id_id", Integer, ForeignKey("patientid.id")),
+    Column("issue_id", Integer, ForeignKey("issue.id"))
+)
+
+
+# think we need to specify some sort of model if we want to 
+# use this table 
+"""
+class PatientIDtoIssue(Base):
+    __tablename__ = "patientidtoissue"
+    patient_id_id = Column(Integer, ForeignKey("patientid.id"))
+    issue_id = Column( Integer, ForeignKey("issue.id"))
+"""
+
+class PatientID(Base):
+    __tablename__ = "patientid"
+    
+    id = Column(Integer, primary_key=True, autoincrement=True)
+    pid = Column(String(50), index=True, nullable=True)
+    ukrdcid = Column(String(50), index=True)
+
+
+class Issue(Base):
```

Comment:
If this is the equivalent of "work item" don't forget we need a way to chain them, so an optional foreign key pointing to another row in the same table, something like `parent`